### ISSUE TITLE
Review code snippet syntax highlighting

### ISSUE THREAD

**Issue by brgrey** [Digital-Humanities-Quarterly/dhq-journal#44]
When code snippets are encoded in articles, language assignment support (e.g., C#, Perl, JSON) for both the `code` and `eg` elements provide syntax highlighting. As part of the general accessibility review, these colors should be reviewed for contrast/legibility. See article [000026](https://github.com/Digital-Humanities-Quarterly/dhq-journal/blob/e23db1c89915f4caa513c18ff30f97c82a8a3456/articles/000026/000026.xml) for an example of automatic highlighting and article [000699](https://github.com/Digital-Humanities-Quarterly/dhq-journal/blob/e23db1c89915f4caa513c18ff30f97c82a8a3456/articles/000699/000699.xml) for an example of manual syntax highlighting.
